packages/style-engine/package.json

```
@@ -1,13 +1,14 @@
 {
 	"name": "@wordpress/style-engine",
-	"version": "0.15.0",
-	"description": "WordPress Style engine.",
+	"version": "1.0.0",
```

```suggestion
	"version": "1.0.0-prerelease",
```

Otherwise it would get bumped to `1.1.0` during npm publishing 😄

---

> Otherwise it would get bumped to 1.1.0 during npm publishing

Ha, good call! Thanks for the tip 😉